listenbrainz/webserver/templates/user/scraper.js

```diff
@@ -449,7 +452,8 @@ function updateLatestImportTimeOnLB() {
         xhr.timeout = 10 * 1000; // 10 seconds
         xhr.onload = function(content) {
             rl_remain = parseInt(xhr.getResponseHeader("X-RateLimit-Remaining"));
-            rl_reset = parseInt(xhr.getResponseHeader("X-RateLimit-Reset"));
+            rl_reset = parseInt(xhr.getResponseHeader("X-RateLimit-Reset-In"));
+            rl_origin = new Date().getTime() / 1000;
```

Comment:
My bad, I'll fix this. In the meanwhile, [LB-195](https://tickets.metabrainz.org/browse/LB-195).